introduction.md

```diff
@@ -0,0 +1 @@
+Hello, this is Beau. I am struggling to understand Git and how to use GitHub. I hope the Code School can help me to become a "GitHub Ninja" and master Git and the command line.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.